oci/main.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import sys
+
+from . import config
+from . import gerrit
+from . import jenkins
+from . import output
+
+
+def run():
+    parser = argparse.ArgumentParser(
+        description='A command line tool for working with Ovirt CI')
+
+    parser.add_argument(
+        '--verbose',
+        help="increase output verbosity",
+        action="store_true")
+    subparsers = parser.add_subparsers(title="commands")
+
+    build_artifacts_parser = subparsers.add_parser(
+        "build-artifacts",
+        help="build artifacts for a change")
+    build_artifacts_parser.set_defaults(command=build_artifacts)
+
+    build_artifacts_parser.add_argument(
+        'change',
+        help='number of the patch to run')
```

Comment:
This sounds like the number of patches we are running, while we build artifacts for a single patch.

How about "Gerrit change number?

Reply:
Done